tests/os/freebsd/snmp/cpu-detailed.robot

```diff
@@ -0,0 +1,49 @@
+*** Settings ***
+Documentation       Check cpu table
+
+Resource            ${CURDIR}${/}..${/}..${/}..${/}resources/import.resource
+
+Test Timeout        120s
+
+
+*** Variables ***
+${CMD}      ${CENTREON_PLUGINS} --plugin=os::freebsd::snmp::plugin
+
+
+*** Test Cases ***
+cpu-detailed ${tc}
+    [Tags]    os    freebsd   cpu
+    ${command}    Catenate
+    ...    ${CMD}
+    ...    --mode=cpu-detailed
+    ...    --hostname=${HOSTNAME}
+    ...    --snmp-version=${SNMPVERSION}
+    ...    --snmp-port=${SNMPPORT}
+    ...    --snmp-community=os/linux/snmp/linux
+    ...    --snmp-timeout=1
+    ...    ${extra_options}
+ 
+    Ctn Run Command And Check Result As Strings    ${command}    ${expected_result}
+
+    Examples:        tc    extra_options                   expected_result    --
+            ...      1     --critical-guest                OK: CPU Usage: user : Buffer creation, nice : Buffer creation, system : Buffer creation, idle : Buffer creation, wait : Buffer creation, kernel : Buffer creation, interrupt : Buffer creation, softirq : Buffer creation, steal : Buffer creation, guest : Buffer creation, guestnice : Buffer creation
```

Comment:
Olivier added a new keyword Ctn Generic Suite Setup to remove the cache betweend each test to be sure the test are reproductible every try.
maybe it's because of the cache you have this output every time in this test.
(if it's true please add the new keyword in every file as it will become a best practice)